zone/exp.cpp

```diff
@@ -801,6 +801,18 @@ void Client::SetEXP(uint64 set_exp, uint64 set_aaxp, bool isrezzexp) {
 		}
 	}
 
+    if (m_pp.exp != set_exp) {
+        const auto exp_value = set_exp - m_pp.exp;
+        const auto export_string = fmt::format("{}", exp_value);
+        parse->EventPlayer(EVENT_EXP_GAIN, this, export_string, 0);
+    }
+
+    if (m_pp.expAA != set_aaxp) {
```

Comment:
These both should be gated with `parse->PlayerHasQuestSub(`

Reply:
Added